utility/servicer/module.go

```diff
@@ -465,6 +465,15 @@ func (s *servicer) executeHTTPRelay(meta *coreTypes.RelayMeta, payload *coreType
 	return &coreTypes.RelayResponse{Payload: string(body)}, nil
 }
 
+// INCOMPLETE(#860): RESTful service relays: basic checks and execution through HTTP calls
+func (s *servicer) executeRESTRelay(meta *coreTypes.RelayMeta, _ *coreTypes.RESTPayload) (*coreTypes.RelayResponse, error) {
+	_, ok := s.config.Services[meta.RelayChain.Id]
+	if !ok {
```

Comment:
```suggestion
	if _, ok := s.config.Services[meta.RelayChain.Id]; {
```

Reply:
@adshmh Noticed you missed this one. It's just an OPTIONAL NIT but let me know what you think about this patter

Reply:
Good catch, thank you. Fixed. I think this improves readability. It is also how other places in this PR perform this check:

![image](https://github.com/pokt-network/pocket/assets/23505281/c6ec990a-2ba6-4a7c-b746-2807b59aabc8)